src/new-client/.env

```diff
@@ -1 +1,2 @@
-NBS_API_BASE_URL=http://localhost:7071
\ No newline at end of file
+NBS_API_BASE_URL=http://localhost:7071
+NBS_CLIENT_BASE_URL=http://localhost:3000
```

Comment:
If NBS_CLIENT_BASE_URL is now required to be set for the auth, I think for this to work when deployed you will need to set this env var in app settings of the app service. This can be found: infrastructure/resources/web.tf > azurerm_linux_web_app > app settings block. Even though this is the the url of itself it should be ok.

Reply:
Aha yes, I had been meaning to speak to you about this actually and check it's okay. I was trying to do it in other ways (thought there might be something native in NextJS, or that we could extract it from the request headers each time or something), but I think this is the most reliable solution (for now at least).

Does `NBS_CLIENT_BASE_URL = "https://${azurerm_linux_web_app.nbs_appts_web_app_service.default_hostname}"` look correct to you? Do we have a prettier hostname alias set up that we should be using instead?

Reply:
Yeah, that looks fine to me. No prettier hostname alias has been set up.